src/fluree/db/flake/flake_db.cljc

```diff
@@ -292,6 +293,131 @@
           (merge-flakes t-new all-flakes)
           (assoc :commit commit-metadata)))))
 
+(defn- component->user-value
+  "Convert an internal pattern component to user-readable format"
+  [db component context]
+  (cond
+    (where/unmatched-var? component)
+    (str (where/get-variable component))
+
+    (where/matched-iri? component)
+    (let [iri (where/get-iri component)]
+      ;; Try to find matching prefix in context
+      (if-let [[prefix ns-iri] (some (fn [[k v]]
+                                       (when (and (string? v) (str/starts-with? iri v))
+                                         [k v]))
+                                     context)]
+        (str prefix ":" (subs iri (count ns-iri)))
+        iri))
+
+    (where/matched-value? component)
+    (where/get-value component)
+
+    (where/matched-sid? component)
+    (let [sid (where/get-sid component db)
+          iri (iri/decode-sid db sid)]
+      (if-let [[prefix ns-iri] (some (fn [[k v]]
+                                       (when (and (string? v) (str/starts-with? iri v))
+                                         [k v]))
+                                     context)]
+        (str prefix ":" (subs iri (count ns-iri)))
+        iri))
+
+    :else
+    (str component)))
+
+(defn- pattern->user-format
+  "Convert internal pattern to user-readable triple format"
+  [db pattern context]
+  (let [ptype (where/pattern-type pattern)
+        pdata (where/pattern-data pattern)]
+    (case ptype
+      :class
+      (let [[s _ o] pdata]
+        {:subject (component->user-value db s context)
+         :property "@type"
+         :object (component->user-value db o context)})
+
+      (:tuple :id)
```

Comment:
This is a small thing, but now that it's user-facing we should rename the `:tuple` pattern (it hasn't been a tuple for a long time) to the `:triple` pattern (or something else that will make sense to a user - SPARQL calls them "basic graph patterns").

Reply:
Addressed in 7555eae3da00a7eed5fad2d035cdbf0f52cc551d